src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -91,11 +93,19 @@ public static function getSubscribedEvents()
     protected function logException(\Exception $exception, $message)
     {
         if (null !== $this->logger) {
-            if (!$exception instanceof HttpExceptionInterface || $exception->getStatusCode() >= 500) {
-                $this->logger->critical($message, array('exception' => $exception));
-            } else {
-                $this->logger->error($message, array('exception' => $exception));
+            $logLevel = LogLevel::ERROR;
```

Comment:
previously was "critical" by default; think that should be kept

Reply:
it was "error" by default, not critical

Reply:
you're right I'll fix

Reply:
Fixed